src/kernel/include/shared/cos_config.h

```diff
@@ -32,6 +32,11 @@
 
 #define COS_MEM_KERN_VA_SZ (1 << 24) /* 16 MB from KERN_START_VA + end of kernel image onward */
 
+#define COS_HW_MMIO_START_VA 0xf0000000 /* Assuming all hardware virtual addresses are beyond this */
```

Comment:
I feel like this should really be MAX SZ of HW MMIO/DMA MAPs and not some fixed VA!. I'll change it to that soon and update.